### PR TITLE
Bl 933 remove avatar from the application process

### PR DESCRIPTION
**Pull request by ezra-casas**
## Description

Removed any association of the AVATAR icon.

Fixes # (issue)
Code Clean up
## Loom Video
(https://www.loom.com/share/1f752d4d972d4a3f9c86df636a084a52)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] I have made corresponding changes to the documentation if necessary (optional)
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Review by Charles-Merrill-Johnson — DISMISSED** [review]
Good job here, pretty simple clean up.

**Review by andrewgomez1992 — DISMISSED** [review]
Nice, easy clean up, I like it!

**Review by Epeng1994 — DISMISSED** [review]
Code looks much cleaner now! Looking great

**Review by stevenhastings — APPROVED** [review]
Wonderful job, Ezra! I guess at this point we just cross our fingers and hope for Auth0 to oblige us, once and for all.  .   . While we're on the subject of Avatars; are we going to talk about the virtual world in the Unreal Engine sitting stage-right in that loom video. . . or, not? My interest is peaked.

**Comment by khaleel737** [comment]
Great job, looks cleaner now

**Review by mcfrr3 — CHANGES_REQUESTED** [review]
Everything looks good. I'm surprised you had to make changes in so many files 😬 😅 . 

I think line 21 in `src/components/pages/Navbar/Navbar.js` is also no longer necessary without the avatar in place

**Review by jack-semder — APPROVED** [review]
I think this is a must needed change, great work!

**Review by theecolya — COMMENTED** [review]
It does not look like you missed anything!

**Review by farhaannishtar — DISMISSED** [review]
Looks great.

**Review by Nicholas-R-Payne — APPROVED** [review]
Good job removing this!

**Comment by mcfrr3** [comment]
> Everything looks good. I'm surprised you had to make changes in so many files 😬 😅 .
> 
> I think line 21 in `src/components/pages/Navbar/Navbar.js` is also no longer necessary without the avatar in place

Sorry, looking back I think it's line 20, where profilePic is being declared. Remove that line and I'll approve the change.

**Comment by Austin-T-Johnson** [comment]
Nice job cleaning up the code everything looks good except you might want to remove line 8 in src/components/common/MemosTable/AddReply/showReply.js as well as line 20 in src/components/pages/Navbar/Navbar.js.
Other than that great job!

File: src/components/common/MemosTable/AddReply/showReply.js [hunk 1]
```diff
@@ -24,19 +24,12 @@ function ShowReply(props) {
...
-            .subtract(2, 'days')
-            .format('YYYY-MM-DD HH:mm:ss')}
+          title={moment().subtract(2, 'days').format('YYYY-MM-DD HH:mm:ss')}
         >
-          <span>
-            {moment()
-              .subtract(2, 'days')
-              .fromNow()}
-          </span>
+          <span>{moment().subtract(2, 'days').fromNow()}</span>
```
Comment (MaxT6) [inline src/components/common/MemosTable/AddReply/showReply.js:32]: nice job refactoring lines 30 and 32 to all be on one line.

**Review by MaxT6 — APPROVED** [review]
Excellent work removing the Avatar from every profile. That should make the profiles clearer to their reviewers.

**Pull request merged**